### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by ruimarinho**
Mostly upgrading babel@7.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,21 +44,21 @@
...
+    "babel-eslint": "^8.0.0",
+    "babel-plugin-istanbul": "^4.1.5",
+    "eslint": "4.11.0",
     "eslint-config-uphold": "0.0.1",
-    "isparta": "^4.0.0",
-    "mocha": "^3.0.2",
-    "nock": "^8.0.0",
-    "nyc": "^8.1.0",
+    "mocha": "^4.0.1",
+    "nock": "^9.0.27",
```
Comment (joaopaulofonseca) [inline package.json]: Maybe use version `9.1.0`?

File: package.json [hunk 2]
```diff
@@ -44,21 +44,21 @@
...
+    "nyc": "^11.2.1",
```
Comment (joaopaulofonseca) [inline package.json]: Maybe use version `11.3.0`?

File: package.json [hunk 3]
```diff
@@ -72,7 +72,7 @@
     "sourceMap": false
   },
   "options": {
-    "mocha": "--compilers js:babel-register --timeout 20000 --recursive --require should"
+    "mocha": "--require @babel/register  --require @babel/polyfill --timeout 20000 --recursive --require should"
```
Comment (joaopaulofonseca) [inline package.json]: Remove extra space before `--require @babel/polyfill`.

File: src/logging/request-obfuscator.js [hunk 4]
```diff
@@ -31,6 +31,10 @@ function obfuscateResponseBody(body, method) {
  */
 
 function obfuscateResponse(request, instance) {
+  if (request.type !== 'response') {
```
Comment (joaopaulofonseca) [inline src/logging/request-obfuscator.js]: Maybe create a separate commits for this and the tests?

Comment (ruimarinho) [reply src/logging/request-obfuscator.js]: Done.

**Pull request merged**